### PR TITLE
[BEAM-3076] support TIMESTAMP in BeamRecordType

### PR DESCRIPTION
**Pull request by zangshayang1**

### CODE REVIEW

**Comment by XuMingmin** [comment]
seems jenkins is not healthy recently. If this build still fails, let's hold for some time. Sorry for the inconvenience.

**Comment by zangshayang1** [comment]
retest this please

**Comment by XuMingmin** [comment]
R: @XuMingmin @xumingming

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +0,0 @@
-Follow this checklist to help us incorporate your contribution quickly and easily:
```
Comment (XuMingmin) [inline .github/PULL_REQUEST_TEMPLATE.md]: please keep the file.

**Comment by zangshayang1** [comment]
Thanks for reviewing @XuMingmin. I restored .github/PULL_REQUEST_TEMPLATE.md.

**Comment by XuMingmin** [comment]
@zangshayang1 I see some tests failed in the jenkins job, can you take a look?
https://builds.apache.org/job/beam_PreCommit_Java_MavenInstall/15409/org.apache.beam$beam-sdks-java-extensions-sql/

**Comment by asfgit** [comment]
SUCCESS 
 
--none--

**Comment by asfgit** [comment]
SUCCESS 
 
--none--

**Comment by asfgit** [comment]
FAILURE
 
--none--

**Comment by aaltay** [comment]
Hey @XuMingmin, any updates on this PR?

**Comment by XuMingmin** [comment]
defer for further discussion

**Pull request closed without merging**